tmt/steps/provision/__init__.py

```diff
@@ -1990,36 +1995,48 @@ class GuestSshData(GuestData):
         default=None,
         option=('-P', '--port'),
         metavar='PORT',
-        help='Use specific port to connect to.',
+        help="""
+             Port to use for SSH connections instead of the default
+             one.
+             """,
         normalize=tmt.utils.normalize_optional_int,
     )
     user: Optional[str] = field(
         default=None,
         option=('-u', '--user'),
-        metavar='USERNAME',
-        help='Username to use for all guest operations.',
+        metavar='NAME',
+        help="""
+             Username to use for SSH connections instead of the account
+             name running tmt.
```

Comment:
By default we're using `root` for basically all provision plugins. What about mentioning this here and removing the duplicated definitions from the individual plugins inheriting from `GuestSshData`?

https://github.com/teemtee/tmt/blob/f413e79d71892f2deacc0d937577799aa3b34e54/tmt/steps/provision/connect.py#L29-L34

https://github.com/teemtee/tmt/blob/f413e79d71892f2deacc0d937577799aa3b34e54/tmt/steps/provision/testcloud.py#L327-L332

Reply:
Indeed, every single plugin has its own default, and it's always `root`. Need to check 1mt plugin, but reducing the boilerplate is good.

Reply:
Nice, thanks!